rel/files/mongooseim.toml

```diff
@@ -256,7 +256,7 @@
   access = "register"
 {{#mod_roster}}
 
-{{{mod_roster}}}
+[modules.mod_roster]{{{mod_roster}}}
```

Comment:
why do we still need templating here if mod_roster is explicitly enabled?

Reply:
`{{#mod_roster}} ... {{/mod_roster}}` is a conditional block that will be inserted only if the `mod_roster` variable is defined.

Reply:
does empty value count as defined or not?
```
{mod_roster, ""}.
```

Reply:
Cause https://mustache.github.io/mustache.5.html specifies only behaviour for an empty list and `false` value

Reply:
`{mod_roster, ""}` counts as defined - module enabled without any option.

Reply:
This conditional expression is already used in many places to generate a nice TOML layout without empty lines. I think we could make it work for other modules work like for `mod_roster`.

Reply:
Thanks for clarification.
I would probably prefer to have a new line in a template file:
```
[modules.mod_roster]
{{{mod_roster}}}
```
A loose a bit of readability in resulting file but have a bit more readable template 
but it can stay like that as well.